### PR TITLE
Require authentication in andi

### PR DESCRIPTION
**Pull request by cameronmarsh**

### CODE REVIEW

File: apps/andi/lib/andi_web/auth/error_handler.ex [hunk 1]
```diff
@@ -0,0 +1,19 @@
+defmodule AndiWeb.Auth.ErrorHandler do
+  @moduledoc false
+  @behaviour Guardian.Plug.ErrorHandler
+
+  require Logger
+
+  @impl Guardian.Plug.ErrorHandler
+  def auth_error(conn, {:unauthenticated, "https://andi.smartcolumbusos.com/roles"}, _opts) do
```
Comment (LtChae) [inline apps/andi/lib/andi_web/auth/error_handler.ex:8]: Any chance we could keep metrics of successful vs failed logins?

File: apps/auth/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Auth
+
+## Installation
```
Comment (LtChae) [inline apps/auth/README.md:3]: We should probably remove the boilerplate

File: apps/auth/lib/auth.ex [hunk 3]
```diff
@@ -0,0 +1,3 @@
+defmodule Auth do
```
Comment (LtChae) [inline apps/auth/lib/auth.ex:1]: What does this do?

**Review by LtChae — APPROVED** [review]

File: apps/auth/README.md (hunk 2 above)
Comment (cameronmarsh) [reply apps/auth/README.md:3]: okay

File: apps/auth/lib/auth.ex (hunk 3 above)
Comment (cameronmarsh) [reply apps/auth/lib/auth.ex:1]: nothing

**Pull request merged**